lib/connection.dart

```diff
@@ -60,7 +60,10 @@ class OldApiBaseAuthConnection implements IOldApiConnection {
         return await loadString(uri, post, contentType, isAuthRelated, retries + 1);
       }
 
-      addCookie(resp.headers.value("set-cookie"));
+      if (resp.cookies.length > 0) {
```

Comment:
`.isNotEmpty` instead of `.length > 0`

Reply:
and still, you could probably omit that `if`, and it'd just `addAll` from the empty list.